### ISSUE TITLE
Call Stack is unreadable in Canary 58

### ISSUE THREAD

**Issue by hutzlibu** [mauricecruz/chrome-devtools-zerodarkmatrix-theme#180]
On chrome it works fine, but in latest canary, it is not nice ...

![bugcanary](https://cloud.githubusercontent.com/assets/7058891/22707167/2acec29c-ed72-11e6-859d-7ad6840b5823.png)

**Comment by mauricecruz** [comment]
Thanks for filing the issue. Will look into it.

**Comment by mauricecruz** [comment]
Setting the native theme to `dark` will fix the styles in the mean time. I'll patch it when I get a chance.

**Comment by arantonius** [comment]
You mean the native theme of chrome?
Well, which one of the countless dark themes, of the store do you mean?
The ones I just tries brought no success so far ... but thanks for the fast reply!

**Comment by hutzlibu** [comment]
You mean the native theme of chrome?
Well, which one of the countless themes from the web store you mean?
The ones I tried brought no success so far, but thanks for the fast reply!

**Comment by mauricecruz** [comment]
![image](https://cloud.githubusercontent.com/assets/1640686/22713789/d423cef2-ed57-11e6-9e71-72fd2a5afa28.png)

**Comment by hutzlibu** [comment]
Erm ... maybe a stupid question, but what is the "native dark theme"?
As far as I know, there are many different dark themes for chrome, but no "native" one from chrome itself.

**Comment by hutzlibu** [comment]
Or other question: which theme do you use?

**Comment by mauricecruz** [comment]
Chrome added a native dark theme a few versions ago. You can enable it in the devtools settings

http://take.ms/Yi9Lp

I still use this theme, but usually use it over the native dark theme because I can't fix some SVG styles.

**Comment by mauricecruz** [comment]
I patched this in the latest update. Please confirm.

**Comment by hutzlibu** [comment]
Now it's dark and readable ... but the mouseover handler for selecting stack elements, is gone ...

p.s. and good to know, that chrome dev tools have finally a native dark theme, but I still prefer your style :)

**Comment by mauricecruz** [comment]
Ahh - I'll fix that. Thanks for the update!

**Comment by mauricecruz** [comment]
Sorry for the delay - should be fixed now!

**Comment by mauricecruz** [comment]
@hutzlibu are we good on this?

**Comment by hutzlibu** [comment]
Sorry, was away.
And I did not really used your extension anymore, since the standard chrome dark style worked. Even though your styles still look nicer. (but I switched chrome-profiles a lot, for debugging and it was annoying to activate everytime again)
 
Anyway, I reinstalled your plugin - on my new chromebook. And the first thing that happened was a complete crash after setting the flag.
But after a restart it worked - but only in combination with the built-in dark style. But this might be just ChromeOS in general, or that I also have a dark theme for ChromeOS in the background.

But I think if it works with the dark style, it is good enough ...
